recipe/meta.yaml

```
@@ -1,47 +1,227 @@
 {% set name = "google-cloud-aiplatform" %}
-{% set version = "1.21.0" %}
+{% set version = "1.66.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-cloud-aiplatform-{{ version }}.tar.gz
-  sha256: 3cd8fdc1b38e2a21552cbdacb338c4779a27c222bb1ea2481eb56b4fa844917b
+  sha256: e95a2b45087e0e2cbfed9ffaf3f10aa8ef3fa2e5642f067246181b5c7b9b2b04
 
 build:
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  # skip s390x: many dependencies missing
+  skip: true  # [py<38 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  entry_points:
+    - tb-gcp-uploader=google.cloud.aiplatform.tensorboard.uploader_main:run_main
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - google-api-core-grpc >=1.32.0,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*
-    - proto-plus >=1.22.0,<2.0.0dev
-    - protobuf >=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
-    - packaging >=14.3,<22.0.0dev
+    - google-api-core >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*
+    - google-api-core-grpc
+    - proto-plus >=1.22.3,<2.0.0dev
+    - protobuf >=3.20.2,<6.0.0dev,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
+    - packaging >=14.3
+    - google-auth >=2.14.1,<3.0.0dev
     - google-cloud-storage >=1.32.0,<3.0.0dev
-    - google-cloud-bigquery-core >=1.15.0,<4.0.0dev
+    - google-cloud-bigquery-core >=1.15.0,<4.0.0dev,!=3.20.0
     - google-cloud-resource-manager >=1.3.3,<3.0.0dev
-    - shapely <2.0.0
+    - shapely <3.0.0dev
+    - docstring_parser <1
+    - pydantic <3
+  run_constrained:
+    # There are a lot of extras for this package and they often use
+    # the same packages with different pinnings. Here follow an approximation
+    # with some of those commented out, and the more restrictive ones
+    # enabled. To allow an easier update of these constraints, I leave
+    # them there and commented out.
+    #
+    # profiler
+    - tensorboard-plugin-profile >=2.4.0,<3.0.0dev
+    - werkzeug >=2.0.0,<2.1.0dev
+    - tensorflow >=2.4.0,<3.0.0dev
+    # tensorboard
+    # - tensorflow >=2.3.0,<3.0.0dev  # [py<312]
+    # metadata
+    # - pandas >=1.0.0
+    - numpy >=1.15.0
+    # xai
+    # - tensorflow >=2.3.0,<3.0.0dev
+    # lit
+    # - tensorflow >=2.3.0,<3.0.0dev
+    # - pandas >=1.0.0
+    - lit-nlp ==0.4.0
+    - explainable-ai-sdk >=1.0.0
+    # - pandas >=1.0.0
+    # - pyarrow >=6.0.1
+    # pipelines
+    - pyyaml >=5.3.1,<7
+    # datasets
+    - pyarrow >=3.0.0,<8.0dev   # [py<311]
+    - pyarrow >=10.0.1          # [py==311]
+    - pyarrow >=14.0.0          # [py>311]
+    # vizier
+     - google-vizier >=0.1.6
```

I'm getting a percy error here and it does look like there's an extra indentation on this line.
(Not sure how it gets through the rest of the system.)
```suggestion
    - google-vizier >=0.1.6
```

---

maybe it has been ignored

---

I will wait for other comments to update this, as tests take long to complete